contracts/src/v0.8/ccip/libraries/Internal.sol

```diff
@@ -81,34 +75,6 @@ library Internal {
     uint256 proofFlagBits;
   }
 
-  /// @notice Report that is submitted by the execution DON at the execution phase.
-  /// @dev RMN depends on this struct, if changing, please notify the RMN maintainers.
-  struct ExecutionReport {
```

Comment:
suggestion: should we rename `ExecutionReportSingleChain` to just `ExecutionReport`, now that we don't have any naming clashes?